### PR TITLE
Feature add signup email confirmation

### PR DESCRIPTION
**Pull request by kbeker**
No issue for that.

### CODE REVIEW

**Comment by kbeker** [comment]
Oh, I see that I forgot to generate migration. I will do it tomorrow.

File: users/views.py [hunk 1]
```diff
@@ -213,3 +227,25 @@ class CustomPasswordResetConfirmView(PasswordResetConfirmView):
...
     template_name = "accounts/password_reset_complete.html"
+
+
+class ActivateAccountView(TemplateView):
+    extra_context = {"MESSAGES": AccountConfirmationText}
+    template_name = "account_confirmation/confirmation.html"
+
+    def activate(self) -> bool:
+        user_id = force_text(urlsafe_base64_decode(self.kwargs["encoded_user_id"]))
+        user = get_user_or_none(user_id=user_id)
```
Comment (dybi) [inline users/views.py]: I would suggest checing if user exist, and `return False` upon `DoesNotExist` exception

File: users/tests/test_views.py [hunk 2]
```diff
@@ -201,3 +207,65 @@ def _get_user_data(self, new_phone_number):
...
+            },
+        )
+
+    def test_signup_view_should_display_signup_form_on_get(self):
+        response = self.client.get(reverse("signup"))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, SignUp.template_name)
+
+    def test_signup_view_should_add_new_user_on_post(self):
+        response = self._register_user_using_signup_view()
```
[the remark this answers is not in the document]
Comment (kbeker) [reply users/tests/test_views.py]: You're right :) Done

File: users/views.py (hunk 1 above)
Comment (kbeker) [reply users/views.py]: Also done :)

**Comment by rwrzesien** [comment]
@kbeker Why not using `all-auth` for this?

**Comment by kbeker** [comment]
@rwrzesien To be honest - I didn't know about this. You think that now it is worth to change it into `all-auth`?

**Pull request merged**